### PR TITLE
Update Maven plugins

### PR DESCRIPTION
**Pull request by arjantijms**

### CODE REVIEW

File: nucleus/parent/pom.xml [hunk 1]
```diff
@@ -155,6 +155,7 @@
         <replacer.plugin.version>1.5.3</replacer.plugin.version>
 
         <easymock.version>4.3</easymock.version>
+        <jacoco.version>0.8.10</jacoco.version>
```
Comment (avpinchuk) [inline nucleus/parent/pom.xml:158]: Property `jacoco.version` already defined at line 1545 for `jacoco` profile. Is that how it should be?

Comment (arjantijms) [reply nucleus/parent/pom.xml:158]: It was exposed outside the `jacoco` profile as well. At least the maven versions plug-in mentioned it.

Comment (avpinchuk) [reply nucleus/parent/pom.xml:158]: Then the definition in line 1545 is not redundant?

Comment (arjantijms) [reply nucleus/parent/pom.xml:158]: I have no idea to be honest. @dmatej ?

Comment (arjantijms) [reply nucleus/parent/pom.xml]: I've removed the property again. Let's look at this in another PR.

**Review by avpinchuk — APPROVED** [review]

**Pull request merged**